UIKitTeamPlate/UIKitTeamPlate/OrderListViewController.swift

```diff
@@ -1,13 +1,340 @@
 // OrderListViewController.swift
 // Copyright © RoadMap. All rights reserved.
+
 // Иван
 import UIKit
 
+/// Переход на корневой контроллер
+protocol Rootable: AnyObject {
+    /// Уведомление, что контроллер только что был закртыт
+    func didDismissModal()
+}
+
 /// Экран счета
-class OrderListViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
-//
-//        // Do any additional setup after loading the view.
-//    }
+final class OrderListViewController: UIViewController {
+    // MARK: - Constants
+
+    enum Constants {
+        static let nameForLeftTraceryImage = "leftTraceryImage"
```

Comment:
```suggestion
        static let nameForLeftTraceryImage = "leftTraceryImage"
```
Image это разве имидж?